packages/types/src/documents/app/view.ts

```diff
@@ -65,7 +65,9 @@ export interface ViewV2 {
   name: string
   primaryDisplay?: string
   tableId: string
-  query?: SearchFilter[]
+  query?: LegacyFilter[] | SearchFilters
+  // duplicate to store UI information about filters
+  queryUI?: SearchFilterGroup
```

Comment:
I was under the impression we would no longer need to go down this road. What happened?

Reply:
Discussed on Discord, the `SearchFilterGroup` is still required for the UI.